internal/provider/provider.go

```diff
@@ -61,7 +68,14 @@ func Provider() *schema.Provider {
 	}
 }
 
+// ListOfAttributesToSkip -> A set of string values, that are being searched and skipped if any found
+// Upon read function. It is set globally, as it does not make sense to skip some attributes over some resources.
+// #YOLO
+var ListOfAttributesToSkip []interface{}
+
 func configureProvider(d *schema.ResourceData) (interface{}, error) {
+	ListOfAttributesToSkip = d.Get("skip_attributes").(*schema.Set).List()
```

Comment:
It's always a good idea to do a nil check on a pointer argument before dereferencing it. It might just work most of the time or if the flow of the program is designed in a way that guarantees the pointer to be initialized, but a nil check is not too expensive anyways.

Reply:
Discussed -> not really necessary at this point